transaction-relayer/src/main.rs

```diff
@@ -168,6 +168,11 @@ struct Args {
     /// How frequently to refresh the address lookup table accounts
     #[clap(long, env, default_value_t = 30)]
     lookup_table_refresh_s: u64,
+
+    /// Space-separated addresses to drop transactions for (OFAC)
+    /// If any transaction mentions these addresses, the transaction will be dropped.
+    #[clap(long, env)]
+    ofac_addresses: Option<String>,
```

Comment:
/// Space-separated addresses to drop transactions for (OFAC)
    /// If any transaction mentions these addresses, the transaction will be dropped.
    #[arg(long, env, value_delimiter = ' ', value_parser = Pubkey::from_str)]
    allowed_validators: Option<Vec<Pubkey>>,